File: Week03/Python/task2.py
print("lütfen 10 ile 100 arasında bir sayı girin")
num=int(input())
fizz_count = 0
buzz_count = 0
fizzbuzz_count = 0

while num<10 or num>100:
    print(" geçersiz sayı,lütfen 10 ile 100 arasında bir sayı girin")
    num = int(input())
x=1
while x<=num:
        if x%7==0:
            x=x+1
            continue
        if x%3==0 and x%5==0:
            print("FizzBuzz")
            fizzbuzz_count += 1


        elif x%3==0:
            print("Fizz")
            fizz_count += 1

        elif x%5==0:
            print("Buzz")
            buzz_count += 1

        else:
            print(x)
        x=x+1
print("--- Summary ---")
print("Fizz count :", fizz_count)
print("Buzz count :", buzz_count)
print("FizzBuzz count:", fizzbuzz_count)

File: Week03/C++/task2.cpp
#include <iostream>
using namespace std;

int main() {

    int n;

    cout << "Lutfen 10 ile 100 arasinda bir sayi giriniz: ";
    cin >> n;

    while (n < 10 || n > 100) {
        cout << "Gecersiz sayi.Lutfen 10 ile 100 arasinda bir sayi giriniz: ";
        cin >> n;
    }

    int fizz = 0;
    int buzz = 0;
    int fizzbuzz = 0;

    for (int i = 1; i <= n; i++) {

        if (i % 7 == 0)
            continue;

        if (i % 3 == 0 && i % 5 == 0) {
            cout << "FizzBuzz" << endl;
            fizzbuzz++;
        }

        else if (i % 3 == 0) {
            cout << "Fizz" << endl;
            fizz++;
        }

        else if (i % 5 == 0) {
            cout << "Buzz" << endl;
            buzz++;
        }

        else {
            cout << i << endl;
        }
    }

    cout << "--- Ozet ---" << endl;
    cout << "Fizz sayisi: " << fizz << endl;
    cout << "Buzz sayisi: " << buzz << endl;
    cout << "FizzBuzz sayisi: " << fizzbuzz << endl;
}